website/content/docs/configuration/client.mdx

```diff
@@ -271,157 +271,6 @@ the full list.
 
 Nomad never attempts to embed the `alloc_dir` in the chroot as doing so would cause infinite recursion.
 
-### `options` Parameters
-
-~> Note: In Nomad 0.9 client configuration options for drivers were deprecated.
-Refer to the [plugin block][plugin-block] documentation for more information.
```

Comment:
These are still deprecated, but they have not been removed. We should not remove this section, as some of the values don't have replacements yet. See https://github.com/hashicorp/nomad/issues/12420